.eslintrc.js

```diff
@@ -5,5 +5,12 @@ module.exports = {
   rules: {
     "@typescript-eslint/ban-ts-comment": 0,
     "license-header/header": [1, "./resources/license-header.js"],
+    "no-use-before-define": [0],
```

Comment:
There's a PR to add these rules to the base lint config. In the meantime, they're duplicated here.

https://github.com/inrupt/javascript-style-configs/pull/10